.github/CODEOWNERS

```diff
@@ -34,6 +34,10 @@
 ## Snaps Team
 /packages/rate-limit-controller            @MetaMask/snaps-devs
 
+## Portfolio Team
+/packages/token-search-discovery-controller/package.json      @MetaMask/portfolio @MetaMask/wallet-framework-engineers
+/packages/token-search-discovery-controller/CHANGELOG.md      @MetaMask/portfolio @MetaMask/wallet-framework-engineers
```

Comment:
I don't know if you saw my last comment on this, but I didn't make the suggestion correctly, sorry.

We can keep these lines as:

``` suggestion
/packages/token-search-discovery-controller      @MetaMask/portfolio
```

but then, can we add these two lines at the bottom?

``` codeowners
/packages/token-search-discovery-controller/package.json      @MetaMask/portfolio @MetaMask/wallet-framework-engineers
/packages/token-search-discovery-controller/CHANGELOG.md      @MetaMask/portfolio @MetaMask/wallet-framework-engineers
```

Reply:
will update here now 👍

Reply:
so in the `## Joint team ownership` specifically?

Reply:
oh no I see that `## Package Release related` makes more sense

Reply:
Yes, that's right, `Package Release related`.